processor/src/main/java/com/linecorp/decaton/processor/runtime/internal/ProcessingContextImpl.java

```diff
@@ -14,7 +14,7 @@
  * under the License.
  */
 
-package com.linecorp.decaton.processor.runtime;
+package com.linecorp.decaton.processor.runtime.internal;
```

Comment:
Shouldn't this class be placed in root/internal ? (i.e. `com.linecorp.decaton.processor.internal`)
As `ProcessingContext<T>` is in root.

Reply:
Exactly. Thanks for pointing out.

Reply:
Resindered, I think no, because while the interface doesn't assume any parts from the runtime package, the Impl requires instances that are of `runtime` package and can't be constructed without that.

Reply:
I see, understood.